.github/tests/prometheus/values.yaml

```diff
@@ -2,3 +2,12 @@ global:
   telemetry:
     prometheus:
       enabled: true
+      podMonitor:
+        enabled: true
+        labels:
+          release: kube-prometheus-stack
```

Comment:
What is the usecase of this label?

Isn't helm already adding a `helm.sh/release` label?

Maybe to avoid confusion like mine add another dummy label if there is no specific reason for this label.

Reply:
kube-prometheus-stack only watches for podmonitors that have labels that match its own labels. So you can have multiple stacks with different labels on the same cluster. It won't be collected without the label.